### PR TITLE
[WIP]: Add support for min-cpu and accelerator

### PR DESCRIPTION
**Pull request by yoanisgil**
This PR adds support for min-cpu-platform and accelerator-type accelerator-count options for the Google driver. 

I'm just sending this to see if there is enough interest to accept the work and get it merged into the main repo.

### CODE REVIEW

File: drivers/google/compute_util.go [hunk 1]
```diff
@@ -274,6 +274,19 @@ func (c *ComputeUtil) createInstance(d *Driver) error {
 		},
 	}
 
+	if len(d.MinCPUPlatform) > 0 {
+		instance.MinCpuPlatform = d.MinCPUPlatform
+	}
+
+	if d.AcceleratorCount != -1 && len(d.AcceleratorType) > 0 {
```
Comment (yoanisgil) [inline drivers/google/compute_util.go]: Need to compare with `0`

**Comment by yoanisgil** [comment]
ping  @dgageo ... it's been a couple of days since I opened the PR and I just wanted to know if this something that could be merged.

**Comment by yoanisgil** [comment]
Anyone?

**Comment by greuff** [comment]
This is exactly what we also need. https://github.com/docker/machine/issues/4582

**Comment by greuff** [comment]
Is this project dead? Still no merge.

**Comment by rakyll** [comment]
Any plans for this PR? This is something I need and happy to help if this PR is abandoned.

**Comment by yoanisgil** [comment]
I don't think this project is actively developed ...

**Comment by greuff** [comment]
Unbelievable that this is still open

**Comment by tmaier** [comment]
This repo is not being supported/maintained anymore.

GitLab is operating a fork. I raised a ticket to get GPU support. See https://gitlab.com/gitlab-org/ci-cd/docker-machine/issues/34

**Comment by stanhu** [comment]
FYI, the GitLab fork of Docker Machine supports this now: https://docs.gitlab.com/runner/executors/docker_machine.html#using-gpus-on-google-compute-engine

**Pull request closed without merging**